packages/components/package.json

```diff
@@ -1,11 +1,18 @@
 {
-  "name": "components",
-  "private": true,
-  "version": "0.0.1",
+  "name": "@portaljs/components",
+  "version": "0.0.3",
   "type": "module",
+  "description": "https://portaljs.org",
```

Comment:
You could probably add a [homepage](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#homepage) here instead of the url, the [description](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#description) is used for searching on npm so you might want to add something that's more indexable to search engines something like "Components for the PortalJS Framework which is a framework for rapidly building rich data portal frontends using a modern frontend approach. Portal.JS can be used to present a single dataset or build a full-scale data catalog/portal."

Reply:
That's true, thank you for this suggestion, I will do that in a future PR